bin/phpliteadmin

```diff
@@ -78,6 +78,11 @@ sed -E -i "s/.*<input[^>]+name='logout'[^>]+>.*//" "$temp/phpliteadmin.php"
 cat > "$temp/phpliteadmin.config.php" <<EOF
 <?php
 
+    // pass PHPSESSID via URL
+    ini_set("session.use_cookies", "0");
+    ini_set("session.use_only_cookies", "0");
+    ini_set("session.use_trans_sid", "1");
+
```

Comment:
Maybe move down below to keep `session.*` together?

Reply:
Done per 31ef211.